cran-comments.md

```diff
@@ -1,12 +1,12 @@
 ## Test environments
-* local Arch Linux Install, kernel 5.16.10-arch1-1, R 4.1.2
-* win-builder (devel and release)
-* r-hub builder
+* local windows 11 install, R 4.3.1
+* github action R-CMD-CHECK: macos-latest (release), ubuntu-latest (devel), ubuntu-latest (oldrel-1), ubuntu-latest (release), windows-latest (release)
+* win-builder.r-project.org (devel)
 
 ## R CMD check results
 There were no ERRORs, no WARNINGs, and 1 NOTEs.
 
-The note is due to resubmitting after a single day. I'm very embarrassed to have to submit this again so quickly. There was a bug that for some reason testthat wasn't picking up due to differences in the environment(). I fixed that, no other changes. My deep apologies for taking up unnecessary time, I thought I had triple-checked everything.
+The note is due to "unable to verify current time" message when verifying file timestamps. From what I can tell, this is due to `check` attempting to access http://worldclockapi.com/, which is no longer available. This note will be resolved if/when the R core team changes this function.
```

Comment:
Oh, would you mind posting this in the #package-maintenance channel of the rOpenSci Slack or on Posit community forum?
This looks like an "old" bug https://stat.ethz.ch/pipermail/r-package-devel/2019q1/003577.html